code/_onclick/hud/alert.dm

```diff
@@ -879,7 +879,6 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		add_overlay(time_left_overlay)
 	if(isnull(poll))
```

Comment:
is this isnull part necessary? should it be moved up to the top? won't it not do anything here at the bottom?

Reply:
Good question. I'm not sure so let's wait for some answers.

Reply:
@SyncIt21 I can confirm that you can remove this isnull(poll) check altogether

Reply:
I'll do that then